ui/pages/Search/VariantSearch.jsx

```diff
@@ -11,8 +11,8 @@ const RESULTS_PATH = 'results/:searchHash'
 const SINGLE_VARIANT_RESULTS_PATH = 'variant/:variantId/family/:familyGuid'
 
 const SEARCH_FORM_PAGES = [
+  'project/:projectGuid/analysis_group/:analysisGroupGuid',
```

Comment:
dynamic analysis groups are not necessarily associated with a specific project which makes the URL for this search page ambiguous as it could be for that group in any project. Therefore, rework that URL to require the relevant project